modules/ROOT/pages/log-search-query-syntax.adoc

```diff
@@ -7,7 +7,7 @@ In addition to using the query builder and facet search user interface in Anypoi
 
 == Search Terms
```

Comment:
So, this looks like a peer with Options Tab. Were they introduced in the topic containing them?
Is this a task: search terms? Or is this a concept? I might have to make Mysti unhappy and suggest that we use gerunds to make that clear: Searching for Terms or Searching Using Terms vs. Terms Used to/for Search.

Search or Search Query?